jenkins-scripts/dsl/gz-collections.yaml

```diff
@@ -365,6 +365,90 @@ collections:
       linux:
         ignore_major_version:
           - gz-jetty
+  - name: 'kura'
+    libs:
+      - name: gz-cmake
+        major_version: 5
+        repo:
+          current_branch: gz-cmake5
+      - name: gz-tools
+        major_version: 2
+        repo:
+          current_branch: gz-tools2
+      - name: gz-utils
+        major_version: 4
+        repo:
+          current_branch: gz-utils4
+      - name: gz-math
+        major_version: 9
+        repo:
+          current_branch: gz-math9
+      - name: gz-plugin
+        major_version: 4
+        repo:
+          current_branch: gz-plugin4
+      - name: gz-common
+        major_version: 7
+        repo:
+          current_branch: gz-common7
+      - name: gz-msgs
+        major_version: 12
+        repo:
+          current_branch: gz-msgs12
+      - name: gz-rendering
+        major_version: 10
+        repo:
+          current_branch: gz-rendering10
+      - name: sdformat
+        major_version: 16
+        repo:
+          current_branch: sdf16
+      - name: gz-fuel-tools
+        major_version: 11
+        repo:
+          current_branch: gz-fuel-tools11
+      - name: gz-transport
+        major_version: 15
+        repo:
+          current_branch: gz-transport15
+      - name: gz-gui
+        major_version: 10
+        repo:
+          current_branch: gz-gui10
+      - name: gz-sensors
+        major_version: 10
+        repo:
+          current_branch: gz-sensors10
+      - name: gz-physics
+        major_version: 9
+        repo:
+          current_branch: gz-physics9
+      - name: gz-sim
+        major_version: 10
+        repo:
+          current_branch: gz-sim10
+      - name: gz-launch
```

Comment:
note that we can exclude gz-launch now or we can do it when bumping major versions.